docker/build/devdeps.manylinux.Dockerfile

```diff
@@ -14,20 +14,18 @@
 # Must be built from the repo root with:
 #   docker build -t ghcr.io/nvidia/cuda-quantum-devdeps:manylinux -f docker/build/devdeps.manylinux.Dockerfile .
 
-ARG manylinux_image=quay.io/pypa/manylinux_2_28_x86_64
+ARG manylinux_image=quay.io/pypa/manylinux_2_28_x86_64:latest
 FROM $manylinux_image
+
 ARG llvm_commit
+ENV CC=/opt/rh/gcc-toolset-12/root/usr/bin/gcc
```

Comment:
Is this set to gcc because of what you mentioned in workflows/deployments.yml ?

> toolchain: gcc12 # set by the dockerfile and not configurable

Reply:
Oh, I forgot to update that - using x86_64 as the toolchain identifier (used in cache keys and image tags) seems appropriate instead for the image. Set to gcc because that was what manylinux comes with, and I didn't look further.